OnionBrowser/Browsing/Tab.swift

```diff
@@ -176,7 +176,19 @@ class Tab: UIView {
 		return refresher
 	}()
 
+	private var _snapshot: UIImage? = nil
 
+	var snapshot: UIImage? {
+			if _snapshot == nil {
+				_snapshot = layer.makeSnapshot(scale: 1.0)
+			}
+			return _snapshot!
+		}
+	
+	func resetSnapshot() {
+		_snapshot = nil
+	}
```

Comment:
I hate this for multiple reasons:
- The code is structured, so methods are below initializers and properties are above.
- Unsymmetric access: Read is a getter, write is a method.

Reply:
I changed position of resetSnapshot(move after initialisers)